ddp-workspace/angular.json

```diff
@@ -1000,4 +1025,4 @@
       "styleext": "scss"
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
Modify our project builds to pick up pepperConfig.js from the directory we generate it into.